datagrid_gtk3/utils/transformations.py

```diff
@@ -105,6 +106,12 @@ def string_transform(value, max_length=None, oneline=True,
                 raise
             value = decode_fallback(value)
 
+    # Remove non-printable caracteres from the string. Only keep those
+    # considered as whitespace/newline. We cannot use string.printable
+    # here as it would remove unicode caracteres too.
+    value = u''.join(c for c in value
+                     if ord(c) >= 32 or c in string.whitespace)
```

Comment:
It would be better to use `is_printable` from `reporting.model` for the condition, it does the same thing -- and if in the future the definition of "printable" changes, it will be in one place. @jcollado is [working](https://github.com/viaforensics/viaextract-main/pull/1741/files#r37523075) to move that into a separate module.